core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java

```diff
@@ -369,7 +369,7 @@ private boolean hasReattachSupportedVolumes(Stack stack, String hostGroupName) {
                 .map(InstanceGroup::getTemplate)
                 .map(Template::getVolumeTemplates)
                 .anyMatch(volumes -> volumes.stream()
-                        .map(VolumeTemplate::getVolumeType).anyMatch(REATTACH_NOT_SUPPORTED_VOLUME_TYPES::contains));
+                        .map(VolumeTemplate::getVolumeType).allMatch(REATTACH_NOT_SUPPORTED_VOLUME_TYPES::contains));
```

Comment:
this is confusing for me, the method name (hasReattachSupportedVolumes) contradicts the last line
```.map(VolumeTemplate::getVolumeType).allMatch(REATTACH_NOT_SUPPORTED_VOLUME_TYPES::contains));```

What does this really do?

Reply:
Renamed